packages/documentation/src/layouts/module-components.js

```diff
@@ -30,13 +43,27 @@ export default class MDXRuntimeTest extends Component {
           <div className="content">
             {children}
             <h2>{data.componentMetadata.displayName}</h2>
-            <p>{data.componentMetadata.docblock}</p>
+            {
```

Comment:
This makes the JsDoc block more readable in the docs. Before this is was 1 line without line breaks.